hip/test/matrix/bccoo_kernels.hip.cpp

```diff
@@ -0,0 +1,407 @@
+/*******************************<GINKGO LICENSE>******************************
```

Comment:
Same as CUDA: remove this whole file as it does not contain any useful code.

Reply:
I have commented the line in hip/test/matrix/CMakeLists.txt

Reply:
Done